overview/glossary.markdown

```diff
@@ -41,74 +103,177 @@ Used to restrict when / where promises are evaluated.
 Appear in front of promises in CFEngine policy, consisting of a class expression followed by two colons.
 Class guards are sometimes called context class expressions.
 
-#### CMDB
+#### Client
+
+In traditional computer networks and software, the client is the program which connects to a server, i.e. the software which initiates the connection in a networked system.
+We say that a server is listening for incoming connections, and servers frequently serve thousands or even millions of clients simultaneously.
+
+In CFEngine, we use the word client to describe all of the hosts which are not hubs.
+A CFEngine hub runs a policy server, which all clients connect to in order to fetch policy.
+
+Historically, the term agent has sometimes been used for this same meaning.
+However, agent also refers to the agent component (the `cf-agent` binary), and thus, when discussing the role of a CFEngine host, _client_ is the preferred term for these hosts which are not hubs, and which packages to install on them.
+
+#### Client initiated reporting
+
+A mode where you change the configuration so that the hub does not initiate connections to client hosts to fetch reports.
+Instead, the clients will establish a connection, and leave it open, until the hub is ready to use it to query for reporting data.
```

Comment:
```suggestion
Instead, the clients will establish a connection, and leave it open for a period of time, giving the hub opportunity to use it to query for reporting data.
```

Reply:
I think mine reads easier, the purpose here in the glossary is to try to explain things in a simple way.